samples/nrf9160/lwm2m_client/src/lwm2m/lwm2m_firmware.c

```diff
@@ -132,6 +134,12 @@ static int firmware_block_received_cb(uint16_t obj_inst_id,
 				"Application");
 	}
 
+	ret = dfu_target_offset_get(&offset);
```

Comment:
Above, there is a comment that says:

```
/* Erase bank 1 before starting the write process */
```

It seems like that could still happen after a board reset, since bytes_downloaded will be 0. Is the comment not accurate?

Reply:
The comment is outdated, I'll remove it.